02-overview-surveys.Rmd

```diff
@@ -15,29 +17,29 @@ When starting a survey, there are multiple things to consider, including the stu
 - Measurement
   - **Validity**: A mismatch between the topic of interest and the question used to collect that information
   - **Measurement Error**: A mismatch between what the researcher asked and how the respondent answered
-  - **Processing Error**: Edits by the researcher to responses provided by the respondent (e.g., adjusts data based on illogical responses)
+  - **Processing Error**: Edits by the researcher to responses provided by the respondent (e.g., adjustments to data based on illogical responses)
 
-Almost every survey will have some errors, and researchers attempt to conduct a survey that reduces this *total survey error*. Weighting is one of the most common ways to reduce total survey error. However, weighting may result in increased error on its own (*adjustment error*). Therefore when designing surveys, researchers should carefully consider ways to reduce each error source. Additionally, when analyzing survey data, understanding decisions that researchers took to minimize these error sources can impact how results are interpreted. The remainder of this chapter dives into considerations for survey development. In these places, we explore how to consider each of these sources of error, and how these error sources can inform the interpretations of the data.
+Almost every survey will have some errors, and researchers attempt to conduct a survey that reduces this *total survey error*. Weighting is one of the most common ways to reduce total survey error. However, weighting may result in increased error on its own (*adjustment error*). Therefore when designing surveys, researchers should carefully consider ways to reduce each error source and total survey error as a whole. Additionally, when analyzing survey data, understanding decisions that researchers took to minimize these error sources can impact how results are interpreted. The remainder of this chapter dives into considerations for survey development. We explore how to consider each of these sources of error, and how these error sources can inform the interpretations of the data.
```

Comment:
Should we add more information about the trade offs w/TSE?  Maybe more about what TSE is in general?